Apps/phredmine/readme.html

```diff
@@ -0,0 +1,21 @@
+<!-- File: readme.html
+  Copyright (c) 2021 Splunk Inc.
+
+  Licensed under Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0.txt)
+-->
+
+<p>
+  <h2>Redmine</h2>
+</p>
+
+<p>Redmine is an open source ticketing system, and the actions performed by these API calls depend to some extend on the configuration on the Redmine instance.</p>
```

Comment:
```suggestion
<p>Redmine is an open-source ticketing system, and the actions performed by these API calls depend to some extent on the configuration of the Redmine instance.</p>
```

Reply:
Please update if the given suggestion is correct.

Reply:
Updated